docs/user_guide/Upgrade_SIMP/Version_Maps/6.4.0_6.5.0.inc

```diff
@@ -74,33 +125,35 @@ In Puppet code:
 Update DNS RNDC key
 """""""""""""""""""
 
-(The following only needs to be done if you are using ``pupmod-simp-named``
-to configure your DNS server and it copies the files using rsync:)
+(The following only needs to be done if you are using
+:package:`pupmod-simp-named` to configure your DNS server and it copies the
+files using rsync:)
 
-On the SIMP Puppet server, remove the ``rndc.key`` from the rsync directories:
+On the SIMP Puppet server, remove the :file:`rndc.key` from the rsync
+directories:
 
 .. code-block:: sh
 
    find /var/simp/environments/ -name rndc.key -exec rm -f {} \;
 
-On your DNS servers, view the ``/etc/rndc.key``.  If the algorithm is
+On your DNS servers, view the :file:`/etc/rndc.key`.  If the algorithm is
 ``hmac-md5``, generate a new rndc key:  Replace ``<keyname>`` with the name
-in the key field in your ``/etc/rndc.key`` files.  (It should be the same as
-the name used in your ``/etc/named.conf``.)
+in the key field in your :file:`/etc/rndc.key` files.  (It should be the same
+as the name used in your :file:`/etc/named.conf`.)
 
 .. code-block:: sh
 
    rndc-confgen -a -k <keyname> -b 256
 
-Set ``svckill::mode`` to ``warning``
-""""""""""""""""""""""""""""""""""""
+Set :code:`svckill::mode` to ``warning``
```

Comment:
should the `` symbol be defined above?

Reply:
I'd considered that, too—but the existing charts described the roles, and it seemed overkill to make a table just for one row of general syntax.

However, if it helps everything look obvious at first glance, I'd be happy to add it to the real style guide (assuming we're happy with using roles for standardized formatting).

Reply:
This is now implemented; you can see it in action at https://simp.readthedocs.io/en/simp-maint-example-role-formatting/contributors_guide/Documentation.html